src/lucky_support/inflector/inflections.cr

```diff
@@ -0,0 +1,142 @@
+module LuckySupport
+  module Inflector
+    extend self
+
+    # TODO: this class needs work to better support adding inflections at runtime, many methods are still missin here.
+    # TODO: this needs a proper spec
```

Comment:
Instead of comments, could you create issues for these so it is easier to track work? It might also be good to make a specific issue for the methods that are missing.

Reply:
I noticed these after the PR and forgot to remove them. Both of those are TODONE ;-)

I like the approach of keeping them as issues though. Kind of has more visibility than in a chance encounter with the particular spot in code.